<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Master</class>
 <widget class="QWidget" name="Master">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="categoryWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 46, 51);
</string>
   </property>
   <widget class="QWidget" name="bookWidget" native="true">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="bookLabel">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>5</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Контакты</string>
     </property>
    </widget>
    <widget class="QWidget" name="bookLine" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>38</y>
       <width>105</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{ 
	background-color: rgb(127, 127, 127);
	max-width:105px; 
	max-height:1px; 
	min-width:105px;
	min-height:1px; 
}
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="storageWidget" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>2</y>
      <width>141</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{ border-radius:10px; background-color: rgb(30, 33, 36);} 


</string>
    </property>
    <widget class="QLabel" name="storageLabel">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>5</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Хранилище</string>
     </property>
    </widget>
    <widget class="QWidget" name="storageLine" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>38</y>
       <width>117</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">

background-color: rgb(255, 0, 127); max-height:1px; min-height:1px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="notesWidget" native="true">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>2</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="notesLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>5</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Заметки</string>
     </property>
    </widget>
    <widget class="QWidget" name="notesLine" native="true">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>38</y>
       <width>95</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{ 
	background-color: rgb(127, 127, 127);
	max-width:95px; 
	max-height:1px; 
	min-width:95px;
	min-height:1px; 
}
</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="toolsWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>530</y>
     <width>420</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 46, 51);</string>
   </property>
   <widget class="QPushButton" name="newButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>5</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ff006c;
 border-style: solid;
 border-radius:20px;
 max-width:40px;
 max-height:40px;
 min-width:40px;
 min-height:40px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../Рабочий стол/ClientRes/add_white.png</normaloff>../../../../../Рабочий стол/ClientRes/add_white.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ff006c;
 border-style: solid;
 border-radius:20px;
 max-width:40px;
 max-height:40px;
 min-width:40px;
 min-height:40px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../Рабочий стол/ClientRes/settings_white.png</normaloff>../../../../../Рабочий стол/ClientRes/settings_white.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>420</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: solid;

</string>
   </property>
   <widget class="QWidget" name="page_3"/>
   <widget class="QWidget" name="page_4"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
